include/crpropa/Vector3.h

```diff
@@ -183,9 +183,9 @@ class Vector3 {
 		return (*this) - getParallelTo(v);
 	}
 
-	// rotate the vector around a given axis by a given a angle
+	// rotate the vector around a given axis by a given angle
 	Vector3<T> getRotated(const Vector3<T> &axis, T angle) const {
-		Vector3<T> u = axis;
+		Vector3<T> u = axis / axis.getR();
```

Comment:
I just noticed that this change apparently make the vector test fail - although I think that a normalized axis here should be reasonable

Reply:
Alright, then we leave getRotated() unchanged for the moment and I change the normalization to the randVectorLamberts() function for now,